dashboard/lib/time.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Compare two time ranges in the format `HH:MM:SS`.
+ */
+export function compareTimeRanges(timeRange1: string, timeRange2: string) {
```

Comment:
[Suggestion] I would convert this to `getTime` comparrison approach:

```
const epoch1 = new Date(timeRange1).getTime()
const epoch2 = new Date(timeRange2).getTime()

return epoch1 - epoch2
```

The only thing I can think of NOT doing so is the `timeRange1` and `timeRange2` are not in a format supported by `Date`, is that the case?